### PR TITLE
Add rustfmt check

### LINKED ISSUE #9: Enforcing rustfmt
We may want to increase maxwidth to 120.

### PR DESCRIPTION
**Pull request by wojciechkepka**
Should fix #9. I've added max_width=120 as @lzutao suggested.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -45,3 +45,11 @@ jobs:
           rustup toolchain install 1.36.0
           rustup default 1.36.0
       - run: cargo build --all-targets --all
+
+  fmt:
+    runs-on: ubuntu-latest
```
Comment (lzutao) [inline .github/workflows/main.yml:50]: There is fmt build job commented. Please use it instead.

**Comment by wojciechkepka** [comment]
> Thanks for the PR.
> What if you creating `.rustfmt.toml` file with the following content:
> 
> ```toml
> use_small_heuristics = "Max"
> merge_derives = false
> max_width = 120
> ```
I've added the config file.

> Also there is already fmt build job in main.yml
Somehow I've missed it. I've removed the new one and uncommented the old one

File: .rustfmt.toml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+use_small_heuristics = "Max"
```
Comment (lzutao) [inline .rustfmt.toml:1]: Could you add `edition = "2018"`? We have just switched to Rust 2018 edition?

**Comment by wojciechkepka** [comment]
I've moved the pull request to #29

**Pull request closed without merging**